core/lib/config/src/configs/btc_watch.rs

```diff
@@ -0,0 +1,55 @@
+use std::time::Duration;
+
+use serde::{Deserialize, Serialize};
+
+/// Configuration for the Bitcoin watch crate.
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
+pub struct BtcWatchConfig {
```

Comment:
Similar to the `EthWatchConfig` we should probably manage here the required confirmations